pkg/cephmgr/osd/partition/scheme.go

```diff
@@ -32,78 +33,96 @@ const (
 	databasePartition     = 2
 	blockPartition        = 3
 	UseRemainingSpace     = -1
-	SimpleVersion         = 1
 	schemeFilename        = "partition-scheme"
+	WalDefaultSizeMB      = int64(576)
+	DBDefaultSizeMB       = int64(20480)
 )
 
-type Scheme struct {
-	Version        int               `json:"version"`
-	SizeMB         int               `json:"sizeMb"`
-	ID             int               `json:"id"`
-	DiskUUID       string            `json:"diskUuid"`
-	PartitionUUIDs map[string]string `json:"partitionUuids"`
+type BluestoreConfig struct {
+	WalSizeMB      int64
+	DatabaseSizeMB int64
 }
 
-// This is a simple scheme to create the wal and db each of size 10% of the disk,
-// with the remainder of the disk being allocated for the raw data.
-func GetSimpleScheme(id, sizeMB int) (*Scheme, error) {
-	diskUUID, err := uuid.NewRandom()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get disk uuid. %+v", err)
-	}
-	walUUID, err := uuid.NewRandom()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get wal uuid. %+v", err)
-	}
-	dbUUID, err := uuid.NewRandom()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get db uuid. %+v", err)
-	}
-	blockUUID, err := uuid.NewRandom()
-	if err != nil {
-		return nil, fmt.Errorf("failed to get block uuid. %+v", err)
+// top level representation of an overall performance oriented partition scheme, with a dedicated metadata device
+// and entries for all OSDs that define where their partitions live
+type PerfScheme struct {
```

Comment:
no there is not another type of scheme anymore.  this is a bit of a naming artifact from when i had both your original Scheme type alongside my new PerfScheme.